rst2pdf/basenodehandler.py

```diff
@@ -103,7 +102,7 @@ def __init__(cls, name, bases, clsdict):
             cls._classinit()
 
 
-class NodeHandler(with_metaclass(MetaHelper, object)):
+class NodeHandler(metaclass=MetaHelper):
```

Comment:
note: Yes, this is correct. You don't need to explicit subclass `object` in Python 3 as discussed [here](https://docs.python.org/3/howto/pyporting.html#subclass-object).